.github/workflows/continuous-integration-unit-tests.yaml

```
@@ -35,4 +35,5 @@ jobs:
     - name: 🔬 Test
       run: |
         yarn test:build:verify
-        yarn test --forceExit
+        yarn test --showConfig
+        TL_LEVEL=info yarn test --expand --detectOpenHandles
```

`--detectOpenHandles` [makes the tests run serially](https://jestjs.io/docs/cli#--detectopenhandles) - it will slow down most of the test suites that can be run in parallelized mode. However it might make no performance difference when run in CI, so this might be ok.
```suggestion
        TL_LEVEL=info yarn test --expand
```